cdk/bin/cdk.ts

```diff
@@ -6,8 +6,12 @@ const app = new App();
 new ManageFrontend(app, 'ManageFrontend-CODE', {
 	stack: 'support',
 	stage: 'CODE',
+	domain: 'manage.code.theguardian.com.origin.membership.guardianapis.com',
```

Comment:
this doesn't follow the `-dev` convention in order to match the certificate domain (which also doesn't have `-dev` in it)